kura/org.eclipse.kura.linux.position/src/main/java/org/eclipse/kura/linux/position/GpsDevice.java

```diff
@@ -314,11 +325,7 @@ public boolean doPollWork() {
 							c = in.read();
 						} catch (Exception e) {
 							s_logger.error("Exception in gps read - {}", e);
-							try {
-								Thread.sleep(1000);
-							} catch (InterruptedException e1) {
-								s_logger.warn("Interrupted - {}", e1);
```

Comment:
@pierantoniomerlino would removing try/catch block around Thread.sleep(1000) here cause the doPollWork() method return true instead of false? In original code, if in.read() throws an exception, we would seep for 1 sec and return false which, in turn, will cause disconnect. Now, if in.read() throws an exception and Thread.sleep(1000), in the catch block throws InterruptedException, it would be caught in the outer catch block and the doPollWork() method returns true; no disconnect will follow. Is this desired behavior?